package.json

```diff
@@ -19,7 +19,7 @@
     "graphql": "yarn babel-node ./scripts/getSchema.js",
     "ios": "yarn react-native run-ios",
     "lint": "yarn jest --config=.jest-eslint.config.js",
-    "postinstall": "node ./scripts/configureApplication.js",
+    "postinstall": "node ./scripts/configureApplication.js && jetify",
```

Comment:
I might be wrong, but I believe `yarn android` jetifies by default.

Reply:
```
$ yarn android
...
info Running jetifier to migrate libraries to AndroidX. You can disable it using "--no-jetifier" flag.
...
```

Reply:
I am not sure it is run when we upload the android framework. I got androidx build errors when trying to run the release script.

Reply:
Ah right... I forgot about this part...